tests/datasets/test_errors_processor.py

```diff
@@ -121,6 +122,7 @@ def test_error_processor() -> None:
                             "build": "3.7.6",
                         },
                         "trace": {"trace_id": trace_id, "span_id": span_id},
+                        "replay": {"replay_id": str(replay_id)},
```

Comment:
feel like i need to add a few more tests cases to this file for my code -- also interesting we only seem to have one? is there some place else my tests should go?

Reply:
Another place to put the test would be `test_replays_api.py`. That writes replays and then reads them. You could also test the case of passing a non-UUID replay in the `replayId` tag.

Reply:
@evanh this is in the errors processor -- would i do `test_errors_api.py`?

Reply:
it seems like that is mainly testing querying, not insertion?

Reply:
You're kind of bumping into some old code that hasn't really changed much, and so has older testing.

Options:
1) Make `test_errors_processing` more modern. Move the test event out of the test and allow it to be customized in different tests (see `test_transactions_processor`). Then you can add tests for your specific case.
2) Leverage some E2E tests as in `test_snql_api` to ensure your data is written able to be read.

Option 1 is the more correct option and if you were on the SnS team I would push you to that.
Option 2 is faster, and I'm OK with it because I don't think it's fair to make product team members upgrade our tests to ship their products (as much as I would like that 😀)